production.md

```diff
@@ -1,58 +1,126 @@
+For example:
 
-So whilst Community will always offer the latest packages as released by maintainers, only by passing these additional quality controls will they make it to Production.
+```r
+install.packages(
+  "polars",
+  repos = c("https://production.r-multiverse.org", getOption("repos"))
+)
+```
+
+## Checks
+
+To reach Production, a package release must pass the following R-multiverse checks:
 
-## A Snapshot Model for Production
+1. `R CMD check` must pass (no errors or warnings) on Mac (R-release), Windows (R-release), Linux (R-devel).
+1. The release must not have an active [security advisory](https://github.com/RConsortium/r-advisory-database).
+1. The `DESCRIPTION` file must not have a `Remotes:` field.
+1. The current version number must be greater than the version numbers of past releases of the same package.
+1. The release must not strongly depend (`Depends:`, `Imports:`, `LinkingTo:`) on an R-multiverse package with any any of the above issues.
+^[However, an R-multiverse package can strongly depend a package from CRAN, regardless of CRAN check status, as long as that package remains available on CRAN.]
 
-A true production experience calls for an alternative distribution model.
-Instead of adopting continuous rolling deployment like CRAN, we provide snapshots of fixed package versions.
+## Snapshots
 
-This has the advantage that:
+Once every 3 months, Production updates all its packages simultaneously and deploys a snapshot.
+Production does not add, remove, or update packages at any other time.
+Packages change slowly in Production, but they are mutually compatible.
+^[And compatible with versions of dependencies that were on CRAN at the time of the snapshot.]
 
-1. All packages within a snapshot are guaranteed to work with each other (at any one point in time on CRAN, there will be packages that fail checks but are within grace periods and have not yet been updated or archived).
-2. Using fixed package versions aids reproducibility.
-3. Does not require constant updates to the latest package versions over the internet.
+## Staging
 
-We take quarterly snapshots on the 15th of each month of February, May, August, and November.
+Rather than pull releases directly from [Community](community.md),
+Production draws from an intermediate repository called [Staging](#staging).
+The [Staging](#staging) repository is active during the month-long period prior to each snapshot.
+During that time, [Staging](#staging) stabilizes the Production candidates
+while still allowing bug fixes.
 
-## The Production Staging Process
+While [Staging](#staging) is active, if a package is failing one or more [R-multiverse checks](#checks),
+then new releases of that package are continuously pulled from [Community](community.md).
+Otherwise, [Staging](#staging) freezes the package at its current release
+and no longer accepts updates from [Community](community.md).
+This freeze prevents new problems in reverse dependencies downstream.
```

Comment:
for such things, later, diagrams would be nice (not now)

Reply:
With a good enough diagram, I agree. Maybe a bit much for this PR.